web/src/mixins/UserDataDBInterface.js

```diff
@@ -5,25 +5,29 @@ export default {
     }
   },
   methods: {
+    getIdeaIndex(ideaId) {
+      return ideaId.replace('C', '').replace('I', '').split('-')[1] - 1;
```

Comment:
Instead of two replace operations, you can do it with regex with `ideaId.replace(/C|I/g, '')`